guides/responsive-material-design-layouts.md

```diff
@@ -72,6 +72,8 @@ children are placed in the content area.
         href="/bower_components/paper-toolbar/paper-toolbar.html">
   <link rel="import" 
         href="/bower_components/iron-flex-layout/iron-flex-layout.html">
+  <link rel="import"
+        href="/bower_components/paper-styles/paper-styles.html">
```

Comment:
Please don't import `paper-styles.html` like this. As mentioned in the docs (https://github.com/PolymerElements/paper-styles/blob/master/paper-styles.html#L33), we recommend importing the needed subfiles, not this generic one